packages/amplication-server/src/core/prismaSchemaUtils/validators.ts

```diff
@@ -13,11 +13,8 @@ export function validateSchemaUpload(file: string): void {
   const schemaString = file.replace(/\\n/g, "\n");
   try {
     validate({ datamodel: schemaString });
-    this.logger.info("Valid schema");
   } catch (error) {
-    this.logger.error(error.message, error, {
-      functionName: "validateSchemaUpload",
-    });
+    console.error(error);
```

Comment:
Do we really want a `console.error()` here and not any logger?

Reply:
Then you will also not need the `eslint-disable-next-line` that we're trying to reduce the use of anyway

Reply:
Yes. The console error shows the error string from Prisma. It is not a must but it is nice to have it